bears/c_languages/CSecurityBear.py

```diff
@@ -0,0 +1,23 @@
+from coalib.bearlib.abstractions.Linter import linter
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+
+
+@linter(executable="flawfinder", output_format="regex",
```

Comment:
flawfinder is a python binary that is istalled at the system. It doesn't work within our virtualenv. Anyone any idea on how we can circumvent this? Making an own flawfinder binary explicitly using /usr/bin/python2.7 to run the original or something?

CC @AbdealiJK